service/routers/users.js

```diff
@@ -0,0 +1,79 @@
+import { Router } from 'express'
+import { User } from '../models/User.js'
+
+const usersRouter = Router()
+
+function sanitizeUsers(users) {
+  const sanitizedUsers = users.map((user) => ({
+    id: user._id,
+    firstName: user.firstName,
+    lastName: user.lastName,
+    email: user.email,
+    decks: user.decks,
+    active: user.active
+  }))
+  return sanitizedUsers
+}
+
+const getUsers = async (req, res) => {
+  const { userId } = req.user
+  const requestor = await User.findById(userId)
+  if (requestor.role === 'admin' || requestor.role === 'superuser') {
+    const users = await User.find({})
+    res.status(201).send(sanitizeUsers(users))
+  } else {
+    res.status(403).send('Forbidden')
+  }
+}
+
+const getUsersById = async (req, res) => {
+  const { userId } = req.user
+  const requestor = await User.findById(userId)
+  if (requestor.role === 'admin' || requestor.role === 'superuser' ||
+    requestor._id.toString() === req.params.id.toString()) {
```

Comment:
If this last logical case is true, do you need to fetch the user again?

Reply:
No i dont need to fetch it again, i removed the line.